pkg/kritis/review/validating_transport.go

```diff
@@ -132,26 +132,11 @@ func (avt *AttestorValidatingTransport) parsePublicKeys(attestorKeys []v1beta1.P
 	return publicKeys, invalidKeys
 }
 
-func (avt *AttestorValidatingTransport) fetchAttestations(image string) ([]*cryptolib.Attestation, error) {
-	atts := []*cryptolib.Attestation{}
-	rawAtts, err := avt.Client.Attestations(image, &avt.Attestor)
+func (avt *AttestorValidatingTransport) fetchAttestations(image string) ([]cryptolib.Attestation, error) {
```

Comment:
In L154, you can just call avt.Client.Attestations directly and get rid of fetchAttestations() altogether.